tests/cypress/e2e/basicE2E.cy.ts

```diff
@@ -0,0 +1,135 @@
+describe('landingRmrk', () => {
+  it('loadLanding', () => {
+    cy.loginWithKeyring()
+  })
+  it('checkNavbar', () => {
+    cy.rmrkNavbar()
+  })
+})
+
+describe('landingBsx', () => {
+  it('loadLanding', () => {
+    cy.visit('/bsx')
+  })
+  it('checkNavbar', () => {
+    cy.bsxNavbar()
+  })
+})
+
+describe('exploreRmrkCollections', () => {
+  it('loadExplore', () => {
+    cy.visit('/rmrk/explore?tab=COLLECTION&page=1')
+  })
+  it('exploreTabs', () => {
+    cy.exploreTabs()
+  })
+  // check collection sort
+  it('collectionsSort', () => {
+    cy.collectionsSortBy()
+  })
+  it('collectionsBuyNow', () => {
+    cy.collectionsBuyNow()
+  })
+})
+
+describe('exploreRmrkGallery', () => {
+  it('loadExplore', () => {
+    cy.visit('/rmrk/explore?tab=GALLERY&page=1')
+  })
+  it('exploreTabs', () => {
+    cy.exploreTabs()
+  })
+  it('expandGallerySearch', () => {
+    cy.expandGallerySearch()
+  })
+  it('galleryInputFields', () => {
+    cy.galleryInputFields(100)
+  })
+  it('gallerySortRmrk', () => {
+    cy.rmrkGallerySortBy()
+  })
+  it('galleryBuyNow', () => {
+    cy.galleryBuyNow(100)
+  })
+})
+
+describe('exploreBsxCollections', () => {
+  it('loadExplore', () => {
+    cy.visit('/bsx/explore')
+  })
+  it('exploreTabs', () => {
+    cy.exploreTabs()
+  })
+  it('collectionsSort', () => {
+    cy.collectionsSortBy()
+  })
+  it('collectionsBuyNow', () => {
+    cy.collectionsBuyNow()
+  })
+})
+
+describe('exploreBsxGallery', () => {
+  it('loadExplore', () => {
+    cy.visit('/bsx/explore')
+    cy.wait(5000)
```

Comment:
hey @kkukelka, thanks for the valuable feedback 👍🏻 

1. I've followed the best practices with `data-cy` and changed almost all selectors to the recommended format
2. There's approx. 4 selectors left with not-very-nice-path, when it comes to generated elements by some of the buefy stuff, I couldn't find a place where to put `data-cy` within our app and still have the element clickable. These parts which need extra attention in the future are marked with `TODO:`
3. I've removed all of the `cy.wait` and solved the occasional issue with giving each test 3 tries to get it right (maybe 2-3 tests sometimes fail because of longer loading time)

Reply:
1. Sweet :)
2. Yeah no worries in some (special) cases it makes little to no sense to use `data-cy` 👍 
3. Sounds like a good solution